pyasic/config/base.py

```diff
@@ -49,7 +49,7 @@ def as_boser(self) -> dict:
         return self.value.as_boser()
 
     def as_epic(self) -> dict:
-        return self.value.as_epic()
+        return self.value.as_epic(self)
```

Comment:
Really curious what the error is when this is removed, I would prefer to fix the underlying issue here rather than pass self...

Reply:
I think I figured this out, you don't need to pass self here, the issue was likely that you didn't call the enum when testing.

`TunerAlgo.voltage_optimizer()` will give you a value that can be converted, `TunerAlgo.voltage_optimizer` is an enum value that doesn't do `as_epic()` properly.

Reply:
I think that makes sense. Heres the simple test I was using for reference.  So I think its converting it somewhere in the code, just need to find where

```
  miner_1 = await get_miner("10.6.0.125")
  config = await miner_1.get_config()
  await miner_1.send_config(config)
```

Reply:
Ok, its definitely loading incorrectly then.  I'll also take a quick look through.

Reply:
I found it, should be good now

Reply:
```
                if algo_info.get("VoltageOptimizer") is not None:
                    return cls.hashrate_tuning(
                        hashrate=algo_info["VoltageOptimizer"]["Target"],
                        algo=TunerAlgo.voltage_optimizer,
                    )
                else:
                    return cls.hashrate_tuning(
                        hashrate=algo_info["ChipTune"]["Target"],
                        algo=TunerAlgo.chip_tune,
                    )
```